tests/unit/querybuilder.spec.ts

```diff
@@ -0,0 +1,9 @@
+import { shallowMount } from '@vue/test-utils';
```

Comment:
Nitpick: can we give the test file the same upper camel case as the component? So `QueryBuilder.spec.ts`.

Reply:
Yeah I should do it.

Reply:
Done!